SecretSanta/src/SecretSanta.Business/IGiftService.cs

```diff
@@ -0,0 +1,16 @@
+﻿using SecretSanta.Data;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+
+namespace SecretSanta.Business
+{
+    public interface IGiftService
```

Comment:
```suggestion
    public interface IGiftService : IEntityService<Gift>
```
Below code can be ignored if IGiftService inherits from IEntityService